src/webgpu/shader/validation/wgsl/runtime-array-not-last-v3.fail.wgsl

```diff
@@ -0,0 +1,8 @@
+# v-0015 - This fails because of the runtime array is not last member of a [[block]] struct.
```

Comment:
This doesn't seem quite right as there is no `struct`, the runtime array is used as the store type?